### ISSUE TITLE
✅ 🐞 Dynamic Link > Lightbox > Video URL vids are now full width

### ISSUE THREAD

**Issue by portrayaloflife** [elementor/elementor#29241]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Lightbox videos used to have an outer margin for better user UI to exit etc, but now they take on full width of the screen with no new options to edit width/height.

### Steps to reproduce

Add a dynamic link > lightbox > and youtube URL to a button. View video lightbox as full width. 

### Expected behavior

Should stay the old size which i believe was width: 85vw; and max-height: 85vh;

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

				
== Server Environment ==
	Operating System: Linux
	Software: Flywheel/5.1.0
	MySQL version: (Google) v5.7.44
	PHP Version: 8.2.24
	PHP Memory Limit: 256M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 900M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.6.2
	Site URL: -
	Home URL: -
	WP Multisite: No
	Max Upload Size: 900 MB
	Memory limit: 256M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Version: 1.2
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 4.8.3
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36

== Active Plugins ==
	Apollo Social Cards
		Version: 5.1
		Author: 

	Astra Pro
		Version: 4.8.3
		Author: Brainstorm Force

	Cloudflare
		Version: 4.12.8
		Author: Cloudflare, Inc.

	Elementor
		Version: 3.25.4
		Author: Elementor.com

	Elementor Pro
		Version: 3.25.2
		Author: Elementor.com

	JetElements For Elementor
		Version: 2.7.1.1
		Author: Crocoblock

	Redirection
		Version: 5.5.0
		Author: John Godley

	Ultimate Addons for Elementor
		Version: 1.36.41
		Author: Brainstorm Force

	User Role Editor Pro
		Version: 4.64.2
		Author: Vladimir Garagulia

	Yoast SEO
		Version: 23.8
		Author: Team Yoast


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Active by default
	Container: Active
	Upgrade Swiper Library: Active by default
	Nested Elements Performance: Active by default
	Optimized Control Loading: Active by default
	Optimized Markup: Inactive by default
	Conditionally load Swiper CSS files: Inactive by default
	Plugin Onboarding: Active by default
	CSS Smooth Scroll: Active by default
	Default to New Theme Builder: Active by default
	Elementor Home Screen: Active by default
	Landing Pages: Inactive
	Nested Elements: Active by default
	Editor Top Bar: Active by default
	Pages Panel: Inactive by default
	Element Caching: Inactive by default
	Link In Bio: Active by default
	Floating Buttons: Active by default
	Elementor Editor Events: Inactive by default
	Atomic Widgets: Inactive by default
	Launchpad Checklist: Inactive by default
	Menu: Inactive by default


== Log ==
	
Log: showing 20 of 372024-08-06 14:31:41 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.19.2',
  'to' => '3.23.3',
)]
2024-08-06 14:31:42 [info] elementor-pro::elementor_pro_updater Started 
2024-08-06 14:31:42 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-08-06 14:31:42 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-08-06 14:31:42 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.19.2',
  'to' => '3.23.3',
)]
2024-11-03 13:16:36 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.23.3',
  'to' => '3.25.2',
)]
2024-11-03 13:16:54 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.23.4',
  'to' => '3.25.3',
)]
2024-11-03 13:17:03 [info] elementor-pro::elementor_pro_updater Started 
2024-11-03 13:17:03 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-11-03 13:17:03 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-11-03 13:17:03 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.23.3',
  'to' => '3.25.2',
)]
2024-11-03 13:17:05 [info] elementor::elementor_updater Started 
2024-11-03 13:17:05 [info] Elementor/Upgrades - _on_each_version Start  
2024-11-03 13:17:05 [info] Elementor/Upgrades - _on_each_version Finished 
2024-11-03 13:17:05 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.23.4',
  'to' => '3.25.3',
)]
2024-11-10 16:45:27 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.3',
  'to' => '3.25.4',
)]
2024-11-10 17:08:14 [info] elementor::elementor_updater Started 
2024-11-10 17:08:14 [info] Elementor/Upgrades - _on_each_version Start  
2024-11-10 17:08:14 [info] Elementor/Upgrades - _on_each_version Finished 
2024-11-10 17:08:14 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.3',
  'to' => '3.25.4',
)]

PHP: showing 3 of 3PHP: 2024-08-06 14:31:37 [error X 2]../wp-content/plugins/elementor-pro/core/app/modules/site-editor/module.php::205] Uncaught Error: Class "ElementorPro\Modules\ThemeBuilder\Module" not found in ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/module.php:205
Stack trace:
#0 ../wp-content/plugins/elementor-pro/core/app/app.php(86): ElementorPro\Core\App\Modules\SiteEditor\Module->__construct()
#1 ../wp-content/plugins/elementor-pro/plugin.php(502): ElementorPro\Core\App\App->__construct()
#2 ../wp-content/plugins/elementor-pro/plugin.php(139): ElementorPro\Plugin->__construct()
#3 ../wp-content/plugins/elementor-pro/plugin.php(542): ElementorPro\Plugin::instance()
#4 ../wp-content/plugins/elementor-pro/elementor-pro.php(68): require('../wp-content...')
#5 ../wp-includes/class-wp-hook.php(324): elementor_pro_load_plugin('')
#6 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#7 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#8 ../wp-settings.php(509): do_action('plugins_loaded')
#9 ../wp-config.php(113): require_once('../wp-s...')
#10 ../wp-config.php(5): require('../wp-config....')
#11 ../wp-load.php(50): require_once('../wp-c...')
#12 ../wp-blog-header.php(13): require_once('../wp-l...')
#13 ../index.php(17): require('../wp-b...')
#14 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-08-06 14:31:44 [warning X 1]../wp-content/plugins/elementor/core/files/manager.php::112] unlink(../wp-content/uploads/elementor/css/post-8078.css): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-08-06 18:02:00 [notice X 4]../wp-content/plugins/elementor-pro/modules/screenshots/module.php::33] Function utf8_decode() is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 2 of 2JS: 2024-08-08 15:30:45 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.23.4:3:1265357] Cannot read properties of null (reading 'getBoundingClientRect') 
JS: 2024-11-03 15:00:56 [error X 3][../wp-content/plugins/elementor/assets/js/frontend.min.js?ver=3.25.3:2:20104] Cannot read properties of undefined (reading 'top') 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified
	JetElements For Elementor: Compatibility not specified
	Ultimate Addons for Elementor: Compatibility not specified

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by portrayaloflife** [comment]
Update: looks like Self hosted videos in the lightbox button link aren't affected and still maintain their padding. It's only youtube video links. Users aren't able to exit the video either due to the bug.

**Comment by portrayaloflife** [comment]
Thank you! Screenshots here as well!

Self Hosted
<img width="1688" alt="image" src="https://github.com/user-attachments/assets/e99f5af9-ba92-4535-9bb5-e4247fa5ac78">

Youtube
<img width="1696" alt="image" src="https://github.com/user-attachments/assets/39534ae3-12f1-40da-93af-5daff9f083a7">

**Comment by hein-obox** [comment]
@nicholaszein I believe that we have caused this bug with the CSS loading improvements. I will try to look into this.

**Comment by portrayaloflife** [comment]
Thank you!!

**Comment by nicholaszein** [comment]
Hey @portrayaloflife!

We have great news! 🙌 

### 📢 We're happy to announce that this issue was resolved with the release of Elementor Pro v3.26.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
